2019-08-20 12:21:19.248367: I tensorflow/core/platform/cpu_feature_guard.cc:141] Your CPU supports instructions that this TensorFlow binary was not compiled to use: SSE4.1 SSE4.2 AVX AVX2 FMA
2019-08-20 12:21:20.095533: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1405] Found device 0 with properties: 
name: Tesla K40c major: 3 minor: 5 memoryClockRate(GHz): 0.745
pciBusID: 0000:84:00.0
totalMemory: 11.92GiB freeMemory: 11.84GiB
2019-08-20 12:21:20.095736: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1484] Adding visible gpu devices: 0
2019-08-20 12:21:27.904756: I tensorflow/core/common_runtime/gpu/gpu_device.cc:965] Device interconnect StreamExecutor with strength 1 edge matrix:
2019-08-20 12:21:27.904991: I tensorflow/core/common_runtime/gpu/gpu_device.cc:971]      0 
2019-08-20 12:21:27.905197: I tensorflow/core/common_runtime/gpu/gpu_device.cc:984] 0:   N 
2019-08-20 12:21:27.905724: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1097] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 11477 MB memory) -> physical GPU (device: 0, name: Tesla K40c, pci bus id: 0000:84:00.0, compute capability: 3.5)
2019-08-20 12:26:40.599986: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1484] Adding visible gpu devices: 0
2019-08-20 12:26:40.600459: I tensorflow/core/common_runtime/gpu/gpu_device.cc:965] Device interconnect StreamExecutor with strength 1 edge matrix:
2019-08-20 12:26:40.600590: I tensorflow/core/common_runtime/gpu/gpu_device.cc:971]      0 
2019-08-20 12:26:40.600718: I tensorflow/core/common_runtime/gpu/gpu_device.cc:984] 0:   N 
2019-08-20 12:26:40.600925: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1097] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 11477 MB memory) -> physical GPU (device: 0, name: Tesla K40c, pci bus id: 0000:84:00.0, compute capability: 3.5)
Traceback (most recent call last):
  File "../exps/tter_test.py", line 158, in <module>
    print_results(result_dicts)
  File "../src/data/t_metrics.py", line 45, in print_results
    idx = np.argmax(result_dicts[run][process_key][metric_key])
UnboundLocalError: local variable 'idx' referenced before assignment
